src/main/java/wootrevived/ifwootaddon/datagen/languages/SimplifiedChinese.java

```diff
@@ -0,0 +1,21 @@
+package wootrevived.ifwootaddon.datagen.languages;
+
+import net.minecraft.data.PackOutput;
+import net.neoforged.neoforge.common.data.LanguageProvider;
+import wootrevived.ifwootaddon.IFWootAddon;
+import wootrevived.ifwootaddon.upgrades.LaserDrill;
+import wootrevived.ifwootaddon.upgrades.MobCrusher;
+import wootrevived.ifwootaddon.upgrades.MobSlaughterFactory;
+
+public class SimplifiedChinese extends LanguageProvider {
+    public SimplifiedChinese(PackOutput output){
+        super(output, IFWootAddon.MOD_ID, "zh_cn");
+    }
+
+    @Override
+    protected void addTranslations() {
+        add(MobCrusher.MOB_CRUSHER_ITEM.get(), "生物粉碎机升级");
+        add(MobSlaughterFactory.MOB_SLAUGHTER_FACTORY_ITEM.get(), "生物屠宰厂升级");
+        add(LaserDrill.LASER_DRILL_ITEM.get(), "镭射钻升级");
```

Comment:
I added one more item to the addon, named "Stygian Laser Drill", add this line here
```java
add(IFItemsRegistry.STYGIAN_LASER_LENS_ITEM.get(), "Stygian Laser Lens");
```
and replace the name within `"` with the chinese one, then if you can run the data generator, if you can't, just commit this simple line change and i'll do it myself.